Gemfile

```diff
@@ -40,3 +40,4 @@ gem "chromedriver-helper"
 gem "launchy"
 gem "poltergeist"
 gem "selenium-webdriver"
+gem "yajl-ruby"
```

Comment:
End with a newline to make the file more convenient for terminal access (e.g. `cat`)